tools/run_tests/run_xds_tests.py

```diff
@@ -311,6 +311,63 @@ def get_client_stats(num_rpcs, timeout_sec):
             logger.debug('Invoked GetClientStats RPC to %s: %s', host, response)
             return response
 
+def get_client_accumulated_stats():
+    if CLIENT_HOSTS:
+        hosts = CLIENT_HOSTS
+    else:
+        hosts = ['localhost']
+    for host in hosts:
+        with grpc.insecure_channel('%s:%d' %
+                                   (host, args.stats_port)) as channel:
+            stub = test_pb2_grpc.LoadBalancerStatsServiceStub(channel)
+            request = messages_pb2.LoadBalancerAccumulatedStatsRequest()
+            logger.debug('Invoking GetClientAccumulatedStats RPC to %s:%d:',
+                         host, args.stats_port)
+            response = stub.GetClientAccumulatedStats(request,
+                                                      wait_for_ready=True,
+                                                      timeout=_CONNECTION_TIMEOUT_SEC)
+            logger.debug('Invoked GetClientAccumulatedStats RPC to %s: %s',
+                         host,
+                         response)
+            return response
+
+def configure_client(rpc_types, metadata):
+    if CLIENT_HOSTS:
+        hosts = CLIENT_HOSTS
+    else:
+        hosts = ['localhost']
+    for host in hosts:
+        with grpc.insecure_channel('%s:%d' %
+                                   (host, args.stats_port)) as channel:
+            stub = test_pb2_grpc.XdsUpdateClientConfigureServiceStub(channel)
+            request = messages_pb2.ClientConfigureRequest()
+            for rpc_type in rpc_types:
+                if rpc_type not in ['empty_call', 'unary_call']:
```

Comment:
Can the rpc_type parameter instead be a list of type `ClientConfigureRequest.RpcType`? Then we wouldn't need this check here or the `if rpc_type == 'empty_call'` conditional in constructing the `request` and `md` protobufs below.

Reply:
Yeah, that's easier. Fix.